cmake/modules/SwiftConfigureSDK.cmake

```diff
@@ -101,6 +102,34 @@ function(remove_sdk_unsupported_archs name os sdk_path architectures_var)
   set("${architectures_var}" ${architectures} PARENT_SCOPE)
 endfunction()
 
+# Work out which threading package to use by consulting SWIFT_THREADING_PACKAGE
+function(find_threading_package sdk default package_var)
+  set(global_override)
+  foreach(elt ${SWIFT_THREADING_PACKAGE})
+    string(REPLACE ":" ";" elt_list "${elt}")
+    list(LENGTH elt_list elt_list_len)
+    if(elt_list_len EQUAL 1)
+      list(GET elt_list 0 global_override)
+      string(TOLOWER "${global_override}" global_override)
+    else()
```

Comment:
Should we have a case to ensure that someone doesn't form an invalid input with multiple `:`?

Reply:
Good idea. Done.